openlibrary/plugins/openlibrary/js/bulk-tagger/BulkTagger.js

```diff
@@ -516,14 +526,20 @@ export class BulkTagger {
         })
             .then(response => {
                 if (!response.ok) {
-                    new FadingToast('Batch subject update failed. Please try again in a few minutes.').show()
+                // Error handling
+                    this.isSubmitting = false;
```

Comment:
```suggestion
```
`!response.ok` implies that an error occurred.
`this.isSubmitting` doesn't do anything.